RFCs/0027-Merge-Dotnet-Test-And-VsTest-Verbs.md

```diff
@@ -0,0 +1,46 @@
+# 0027 Merge dotnet test and dotnet vstest verb
+
+## Motivation
+Customer ask to merge the two dotnet verbs "test" and "vstest".
+Associated github issue: https://github.com/Microsoft/vstest/issues/1453
+
+## Goal
+User should be able to run unit and integration tests with ability to pass in the filename of either a .csproj or a .dll without having to change any other syntax of the dotnet test command.
+
+## Success criteria (as per user ask)
+1. The "dotnet test" command can be used in place of the "dotnet vstest" command.
+2. The "dotnet test" command can accept a list of dlls
+3. Using "dotnet test" and targeting a list of dlls, there is an argument for "--parallel" which will run the tests from the dlls in parallel.
+4. Using "dotnet test" and targeting a dll, the inclusion or exclusion of the "--no-build" argument will have no impact on the behavior of execution.
+
+## Issues with merging the two verbs
+
+### Irrelevant Arguments
+The arguments that are acceptable for the dotnet test for project/solution will not be applicable to the dotnet test when invoked for dlls.
+Here are possible ways to handle this:
+
+1. Ignore the arguments that are not valid.
+2. Throw a warning to the user, and continue with the execution.
+3. Throw an error, giving user an actionable message and stop the execution.
```

Comment:
I think option 2 makes most sense. It should not fail the execution but ignoring the argument doesn't feel good either.